### PR TITLE
[WKINT-396] Fix tests that are hanging in CI

### PR DESCRIPTION
**Pull request by derekwbrown**
There are a fewproblems which may be fixed by this.

This PR fixes the fact that USM was broken.  Also fixes the intermittent that was failing the tests.

Fixes problem in underlying ETW code that was causing tests to 
intermittently fail.

Underlying problem is that the ETW message that was being used to
clean up a connection sometimes comes out of the expected order (
the close comes before the connection sent notificiation).  This was
causing us to intermittently drop a connection.  In the real world,
it would mean that every now and again a connection would be missed.

In the test, when that one case came up, the test would fail and
hang forever.

This PR fixes the underlying problem by switching the ETW notification that causes cleanup to a different message, which (empirically) always comes last.

Also add additional test that floods the web server.  With the 10k connection test, (empirically again) I always got the out of order at least once.
Of course, since it's not predictable, it's possible to do 10K conns and always get the original behavior.  But the change in the test should cover the out-of-order case and prove that it's fixed.

Expanding the test yielded another problem:  we can actually exceed IIS' capacity by flooding the server in this manner.
Add the ability to track which ETW IDs have been received for a given connection.   This is for present and future debugging.

For the flood test, expect that not all connections will succeed, and count accordingly.

### CODE REVIEW

**Review by richl-dd — APPROVED** [review]

File: test/kitchen/test/integration/win-sysprobe-test/rspec_datadog/win-sysprobe-test_spec.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ def check_output(output, wait_thr)
```
Comment (pducolin) [inline test/kitchen/test/integration/win-sysprobe-test/rspec_datadog/win-sysprobe-test_spec.rb:26]: **💬 suggestion**
Can you please add WKIT as owners of this file? It is currently owned by Agent Platform, not sure why 
image

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -511,6 +511,7 @@
 /test/kitchen/test/integration/win-install-fail/ @DataDog/windows-agent
 /test/kitchen/test/integration/win-installopts/ @DataDog/windows-agent
 /test/kitchen/test/integration/win-no-subservices/ @DataDog/windows-agent
+test/kitchen/test/integration/win-sysprobe-test/ @DataDog/windows-kernel-integrations
```
Comment (pducolin) [inline .github/CODEOWNERS:514]:
```suggestion
/test/kitchen/test/integration/win-sysprobe-test/ @DataDog/windows-kernel-integrations
```

**Review by pducolin — APPROVED** [review]

**Review by bkabrda — APPROVED** [review]

**Pull request merged**